### DISCUSSION TITLE
I'm having issues with PKG update and PKG upgrade in termux.

### DISCUSSION THREAD

**Discussion by Rehan587** [termux/termux-packages#8857] [Q&A] [answered]
~$ termux-info
Application version:
0.117
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://termux.librehat.com/apt/termux-main stable main
# sources.list.d/hax4us_pentesting.list
deb https://hax4us.github.io/termux-x/ pentesting main
# sources.list.d/gushmazuko.list
deb https://github.com/gushmazuko/metasploit_in_termux/raw/master gushmazuko main
Updatable packages:
All packages up to date
Android version:
11                                                                                                                                  Kernel build information:
Linux localhost 4.14.186-g4f65b02ee-dirty #1 SMP PREEMPT Thu Dec 16 18:28:32 CST 2021 aarch64 Android                               Device manufacturer:
vivo
Device model:
vivo 1915

![Screenshot_2022_0131_223914.jpg](https://user-images.githubusercontent.com/84271835/151841511-ce3e5d79-f363-4612-8e7a-dc93fc58ff2f.jpg)

Pls help 🆘

**Comment by xtkoba** [comment, marked as answer]
#8856
